gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
`require 'earlgrey/cli'` can load from gems `require_relative` can't

we don't want the spec loading from the earlgrey installed gem. I don't think we have any guarantees what the $LOAD_PATH looks like.